packages/ag-charts-community/src/chart/series/cartesian/barAggregation.ts

```diff
@@ -129,8 +107,22 @@ export function computeBarAggregation(
         throw new Error('Negative aggregation data missing in split mode');
     }
 
-    let positiveIndices = getIndices(maxRange, positiveIndexData);
-    let negativeIndices = getIndices(maxRange, negativeIndexData);
+    let positiveIndices = getMidpointsForIndices(
+        maxRange,
+        positiveIndexData,
+        undefined,
+        undefined,
+        undefined,
+        existingFilter?.positiveIndices
+    );
```

Comment:
Is it worth re-ordering these params to not have to add the `undefined` values, or is it commonly used with those params elsewhere?